mehalsgmues/templates/home.html

```diff
@@ -35,6 +35,13 @@ <h2 id="mapjobs">
         {% endif %}
     {% endif %}
     {#% include "mag/snippets/price_change.html" %#}
+
+    <h2 class="wp-title">{% trans "EAT Zähler" %}</h2>
```

Comment:
I placed it above the news now but I'm not sure whether it would better to be placed below...